packages/react/src/drawer/docs/overview.mdx

```diff
@@ -347,3 +347,65 @@ By default, Drawer’s overlay defaults to 80% opacity, however, by adding `mute
 	);
 };
 ```
+
+## Focus custom element on close
+
+When a drawer closes, focus is returned to the element that opened it. You can override this behaviour by passing an element to the `elementToFocusOnClose` prop. This can be used in situations, for example, when an action performed in the drawer causes an error, setting the main page in an error state, and requiring the user’s focus to be returned to the alert instead.
```

Comment:
@stowball I've updated this based on our conversation. Let me know what you think.

Reply:
Looks good to me!